RocketChat/Rocket.Chat
# [FIX] Unread bar in channel flash quickly and then disappear

Co-authored-by: gabriellsh <40830821+gabriellsh@users.noreply.github.com>







## Proposed changes (including videos or screenshots)


![unread_messages](https://user-images.githubusercontent.com/27704687/121092865-960dc600-c7c2-11eb-9074-81060d826811.gif)


## Issue(s)


## Steps to test or reproduce

- Log as user A and open channel A
- Log as user B, open channel B, and send a bunch of messages enough to show the unread messages bar
- Open the user A view and check the unread messages bar on channel B

## Further comments
